src/firebase-namespace.ts

```diff
@@ -46,6 +47,24 @@ const DEFAULT_APP_NAME = '[DEFAULT]';
  */
 export const FIREBASE_CONFIG_VAR: string = 'FIREBASE_CONFIG';
 
+/**
```

Comment:
I'm a bit uneasy about this logic residing in the global firebase-namespace file. Can we somehow move this to the database module?